app/forms/search_keywords_form.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+class SearchKeywordsForm < ApplicationForm
+  attr_reader :keywords, :search_params, :pagination_params
+
+  # rubocop:disable Lint/MissingSuper
```

Comment:
Do we really need this disabled? 🤔

Reply:
The lint is unrelated, and it is annoying so I decided to disable it.

Reply:
Why not add `super` at the end of your constructor? 🤔 
By default this will pass the arguments, if you do not have any argument you can add `super()`. 

The use case is if later `ApplicationForm` changes and must be initialized, we must not skip its constructor. In this context, it is not a big deal, as we would just "fix the code", but if `ApplicationForm` was declared in a GEM, and a new version must initialize something (with no arguments to avoid breaking changes), then we might get an error.